playwright-e2e/tests/dsm/tissue-request/cmi-tissue-request.spec.ts

```diff
@@ -119,26 +153,235 @@ test.describe.skip('Tissue Request Flow', () => {
+
+        //Now in Tissue Request page - check the input for the main Tissue Request Section
+        const faxSentDate = await tissueInformationPage.getFaxSentDate(0);
+        expect(faxSentDate).toBe(today);
+
+        const tissueReceivedDate = await tissueInformationPage.getTissueReceivedDate();
+        expect(tissueReceivedDate).toBe(today);
+
+        const tissueReqestNotes = await tissueInformationPage.getNotes();
+        expect(tissueReqestNotes).toBe(testRequestNotes);
+
+        //Check the input for the tissue block
+        tissue = tissueInformationPage.tissue();
+
+        const ussUnstainedValue = await tissue.getFieldValue(Label.USS_UNSTAINED);
+        expect(ussUnstainedValue).toBe(testMaterialsReceivedValue);
+
+        const blockValue = await tissue.getFieldValue(Label.BLOCK);
+        expect(blockValue).toBe(testMaterialsReceivedValue);
+
+        const heValue = await tissue.getFieldValue(Label.H_E_PLURAL);
+        expect(heValue).toBe(testMaterialsReceivedValue);
+
+        const scrollValue = await tissue.getFieldValue(Label.SCROLL);
+        expect(scrollValue).toBe(testMaterialsReceivedValue);
+
+        //TODO check that the SMIDs have been saved
+
+        const expectedReturnDate = await tissue.getFieldValue(Label.EXPECTED_RETURN_DATE);
+        expect(expectedReturnDate).toBe(today);
+
+        const returnDate = await tissue.getFieldValue(Label.RETURN_DATE);
+        expect(returnDate).toBe(today);
+
+        if (isClinicalStudy) {
+          const dateSentForExternalPathReview = await tissue.getFieldValue(Label.DATE_SENT_FOR_EXTERNAL_PATH_REVIEW);
+          expect(dateSentForExternalPathReview).toBe(today);
+
+          const dateReceivedFromExternalPathReview = await tissue.getFieldValue(Label.DATE_RECEIVED_FROM_EXTERNAL_PATH_REVIEW);
+          expect(dateReceivedFromExternalPathReview).toBe(today);
+        }
       });
 
-      await test.step('Deleting OncHistory tab row', async () => {
+      await test.step('Go back to participant list to add External Path Review dates and confirm they appear in Ptp List Download', async () => {
+        if (isClinicalStudy) {
+          //Go back to Pariticpant List
+          await tissueInformationPage.backToParticipantList();
+          await participantListPage.waitForReady();
+
+          //Add the following columns to the Participant List: Date Sent For External Path Review, Date Received From External Path Review
+          const customizeViewPanel = participantListPage.filters.customizeViewPanel;
+          await customizeViewPanel.open();
+          await customizeViewPanel.selectColumns(
+            'Tissue Columns',
+            [Label.DATE_SENT_FOR_EXTERNAL_PATH_REVIEW, Label.DATE_RECEIVED_FROM_EXTERNAL_PATH_REVIEW]
+          );
+          await customizeViewPanel.close();
+
+          //Download the Participant List
+          const download = await participantListPage.downloadParticipant({ fileFormat: FileFormat.XLSX });
```

Comment:
this handy part is pretty much from the `download-all-data.spec.ts` test - modified very slightly for this test